scm/src/main/java/com/sandboni/core/scm/resolvers/cli/GitDiffRunner.java

```diff
@@ -69,16 +69,22 @@ private static boolean isNewDiff(String line) {
         command.add("git");
         command.add("diff");
         command.add("-U1");
-        command.add(fromRevision.getName());
+        command.add(toRevision.equals(fromRevision)? fromRevision.getName() + '~' : fromRevision.getName());
+
         if (!toRevision.equals(ObjectId.zeroId())) {
             command.add(toRevision.getName());
         }
+
+        buildDiffCommandHelper(command);
```

Comment:
Can we please change the name of this method to something like "addFileTypeFilters"?